.travis.yml

```diff
@@ -1,35 +1,27 @@
 language: python
 sudo: false
 python:
-- 2.6
 - 2.7
-- 3.2
-- 3.3
-- 3.4
-- 3.5
+- 3.6
 - pypy
 - pypy3
 env:
 - secure: "EhG2uSD2m1eGxuL2HeQewJJx7MVL4WpjrxyfiUBEgsApemD1yKJPjUnLwAyd\nbPi5HJx5ySm1TTRSvj6/yP85YLYTaJHA8OabKk7p0wFW294qcrYIDGovU7NL\n3YOqZmqN+S3XOBGFCOnByxE+pcxxWW/3/I09EgeW7D6tBPh67G0="
 install:
 - echo $TRAVIS_PYTHON_VERSION
-- pip install pytest pytest-xdist pytest-capturelog memory_profiler==0.40 psutil coveralls flake8
+- pip install pytest pytest-xdist pytest-catchlog memory_profiler==0.40 psutil coveralls
```

Comment:
FYI [since the version 3.3.0, pytest has its own built-in catchlog/caplog feature.][1]

[1]: https://docs.pytest.org/en/latest/changelog.html#pytest-3-3-0-2017-11-23

Reply:
Ah. So it's safe to remove the catchlog dependency. Hmm. I think that was also pointed out in a previous attempt to rework CI.